### DISCUSSION TITLE
Cannot connect to specific peers and they cannot connect to me

### DISCUSSION THREAD

**Discussion by lsherring** [Chia-Network/chia-blockchain#9142] [Support]
Since I had to start from scratch, I'm trying to get to some super nodes (good speed) I know of.  I cannot connect and there is zero in the log file.  They connect to me they connect but immediately disconnect in a fraction of a second on their end.  There is also nothing in the other users log as well.  I get all the rogue IPs always connecting and am always playing whackamo with it but I'm not getting any good quality nodes.  I cannot connect regardless of what I try.

**To Reproduce**

Connect to peers via the gui. I cant telnet as I don't have the pwd but it does try and ping works just fine. 

Running 1.2.10 and I have a decent Rig with a Ryzen 3950  processor.

**Expected behavior**

**Screenshots**

**Desktop**

- OS: 
- OS Version/Flavor: 
- CPU: 

**Additional context**

**Comment by emlowe** [comment]
Peers have a maximum number they support, so it's likely the peer you are trying to add is just full - so they kick you out. You should just let the network find peers for you, it will eventually find good peers to sync from

**Reply by lsherring** [reply]
Respectfully disagree.  I worked with others on the Chia Forum for a few days.  We all deleted our rogue peers.  None of us were ever full when we tried to connect to each other.  The problem still exists.  I get many nodes that just sit there and do nothing.  Nothing up or down.  LAstly, with 50% availability peer-wise, no one connects to me and I cannot connect to them.  This is still an issue.
